core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -512,24 +513,28 @@ public List<String> getManagerLocations() {
     OpTimer timer = null;
 
     if (log.isTraceEnabled()) {
-      log.trace("tid={} Looking up manager location in zookeeper.", Thread.currentThread().getId());
+      log.trace("tid={} Looking up manager location in zookeeper at {}.",
+          Thread.currentThread().getId(), zLockManagerPath);
```

Comment:
Some of these log message changes seem unrelated to this PR. Might be useful to do those separately, first, so they don't detract from reviews on the main changes intended in this PR.